guides/source/active_job_basics.md

```diff
@@ -499,14 +694,14 @@ can be used to find out if a given job was successfully enqueued.
 
 ### Queue Backend Support
 
-For `perform_all_later`, bulk enqueuing needs to be backed by the [queue
-backend](#backends).
+For `perform_all_later`, bulk enqueuing needs to be backed by the queue backend.
+Solid Queue, the default queue backend, supports bulk enqueuing using
+`enqueue_all`.
 
-For example, Sidekiq has a `push_bulk` method, which can push a large number of
-jobs to Redis and prevent the round trip network latency. GoodJob also supports
-bulk enqueuing with the `GoodJob::Bulk.enqueue` method. The new queue backend
-[`Solid Queue`](https://github.com/rails/solid_queue/pull/93) has added
-support for bulk enqueuing as well.
+[Other backends](#alternate-queuing-backends) like Sidekiq have a `push_bulk`
```

Comment:
Question for consideration/discussion - or maybe one that needs more thought and to be dealt with at a later date.

Does it dilute the guide too much to start going down the path of alternate backends? I considered this question a few times when working on Testing. It's a guide that's really about minitest, so I landed on taking out the few RSpec mentions, with the view to perhaps offering an alternate guide sometime in the future. Your call - but just thought it might be an interesting one to consider.

Reply:
This is a good point @OughtPuts  and one that we can continue to discuss.

My opinion on this is that it’s fine to mention them where relevant, especially since not everyone may be using the default backend yet given how new it is. Over time, we can gradually phase out other references and consolidate everything into one place if necessary.